### PR TITLE
Refactor CurationGrid to use css prop

### PR DESCRIPTION
**Pull request by archibalderson**
Resolves #NUMBER

**Overall change:**
Refactored CurationGrid to use css prop 

**Code changes:**

- replaced styled api with css prop
- moved styles to index.styles

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by rebeccamcginn** [comment]
Testing:

Compare with a standard live Topic page (https://www.bbc.com/pidgin/topics/c95y35941vrt)
This is a refactor; the styling on this branch should match the styling on live.
Verify across breakpoints

File: src/app/pages/TopicPage/Curation/CurationGrid/index.styles.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      verticalAlign: 'top',
+      display: 'inline-block',
+      marginBottom: `${spacings.DOUBLE}rem`,
+      width: '100%',
+      [mq.GROUP_2_MAX_WIDTH]: {
+        borderTop: '1px #e6e8ea solid',
+        paddingTop: `${spacings.FULL}rem`,
+      },
+      [mq.GROUP_3_ONLY]: {
+        marginInlineEnd: `${spacings.DOUBLE}rem`,
```
Comment (amoore108) [inline src/app/pages/TopicPage/Curation/CurationGrid/index.styles.jsx:35]: Nice! Good to see the use of these directional css properties :)

**Review by amoore108 — APPROVED** [review]
Great work!

**Review by rebeccamcginn — APPROVED** [review]
We can fix this merge conflict today if you're free 😄

**Pull request merged**